### PR TITLE
i18n(fr): Update `guides/integrations-guide/sitemap.mdx` from #8840

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
Update `guides/integrations-guide/sitemap.mdx` from #8840

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/fr/guides/integrations-guide/sitemap.mdx [hunk 1]
```diff
@@ -401,6 +401,22 @@ Le plan du site qui en résulte ressemble à ceci :
 ...
 ```
 
+### xslURL
+L'URL d'une feuille de style XSL pour styliser et embellir votre sitemap.
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/guides/integrations-guide/sitemap.mdx:405]: nit:
```suggestion

L'URL d'une feuille de style XSL pour styliser et embellir votre sitemap.
```

File: src/content/docs/fr/guides/integrations-guide/sitemap.mdx [hunk 2]
```diff
@@ -401,6 +401,22 @@ Le plan du site qui en résulte ressemble à ceci :
...
+La valeur définie peut-être soit un chemin relatif à votre URL configurée `site` pour une feuille de style locale, soit un lien URL absolu vers une feuille de style externe.
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/guides/integrations-guide/sitemap.mdx:406]: Nit for the empty line. The wording also seems a bit strange to me:
* "à votre URL configurée `site`" sounds weird
* "lien URL": I think in French we use either "lien" or "URL", I don't think I've ever read "lien URL".

What do you think of:
```suggestion

La valeur définie peut être soit un chemin relatif à l'URL configurée pour votre `site` dans le cas d'une feuille de style locale, soit une URL absolue vers une feuille de style externe.
```

**Review by ArmandPhilippot — CHANGES_REQUESTED** [review]
Some nitpicking about the blank lines. Since they are in the original version, we might as well keep them so the line numbers match. I also suggested a small rewording.

**Review by ArmandPhilippot — APPROVED** [review]
I forgot about those two line breaks... but I'm nitpicking. So whether you add them or not, LGTM!

**Comment by thomasbnt** [comment]
> I forgot about those two line breaks... but I'm nitpicking. So whether you add them or not, LGTM!

Thanks again @ArmandPhilippot for your review!

**Review by yanthomasdev — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**